make/barman/make.sh

```diff
@@ -1,11 +1,13 @@
 echo ">>> Making barman"
 
-for VALS in "BARMAN_VERSION=2.3 BARMAN_PACKAGE_VERSION=2.3-2.pgdg80+1 PG_CLIENT_MAJOR_VERSION=9.6 PG_CLIENT_VERSION=9.6 PG_CLIENT_PACKAGE_VERSION=9.6.7-1.pgdg80+1" \
-            "BARMAN_VERSION=2.3 BARMAN_PACKAGE_VERSION=2.3-2.pgdg80+1 PG_CLIENT_MAJOR_VERSION=10.2 PG_CLIENT_VERSION=10 PG_CLIENT_PACKAGE_VERSION=10.2-1.pgdg80+1"; do
+for VALS in "BARMAN_VERSION=2.3 BARMAN_PACKAGE_VERSION=2.3-2.pgdg80+1"; do
```

Comment:
redundant for

Reply:
I leave this for-loop for future version of barman.
And this script is really not need to concern for performance.